docs/labs/sending_output.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html><script src="template_files/page-script.js" id="bw-fido2-page-script"></script><head>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="template_files/style.css">
+<link rel="stylesheet" href="template_files/checker.css">
+<script src="template_files/js-yaml.min.js"></script>
+<script src="template_files/checker.js"></script>
+<link rel="license" href="https://creativecommons.org/licenses/by/4.0/">
+
+<!-- See create_labs.md for how to create your own lab! -->
+
+<!-- Sample expected answer -->
+<script id="expected0" type="plain/text">
+private final List<String> allowedHosts = new String[]{"127.0.0.1"}
+</script>
+<!--
+-->
+<script id="expected1" type="plain/text">
+private final List<String> allowedHosts = new String[]{"localhost"}
+</script>
+
+<!-- Full pattern of correct answer -->
+<script id="correct0" type="plain/text">
+private final List<String> allowedHosts = new String\[]{"(127\.0\.0\.1|localhost)"}
+</script>
+
+<script id="correct1" type="plain/text">
+private final List<String> allowedHosts = new String\[]{"127\.0\.0\.1",(\s)*"localhost"}
+</script>
+
+<script id="correct2" type="plain/text">
+private final List<String> allowedHosts = new String\[]{"locahost",(\s)*"127\.0\.0\.1"}
+</script>
+
+<script id="info" type="application/yaml">
+---
+hints:
+- present: "new String[]{}"
+  text: Add allowed hosts the string array enclosed in curly brackets.
+- present: "private final List<String> allowedHosts ="
+  text: Make sure that the allowedHosts constant is an Java inline string array.
+# debug: true
+</script>
+</head>
+<body>
+<!-- For GitHub Pages formatting: -->
+<div class="container-lg px-3 my-5 markdown-body">
+<h1>Format Strings and Templates Lab</h1>
+<p>
+This is a lab exercise on developing secure software.
+For more information, see the <a href="https://best.openssf.org/labs/introduction.html" target="_blank">introduction to
+the labs</a>.
+
+</p><p>
+</p><h2>Task</h2>
+<p>
+<b>Restrict the JNDI hostnames from which variables can be loaded.</b>
+
+</p><p>
+</p><h2>Background</h2>
+<p>
+In this exercise, we'll assume that out output template allows a user to specify a JNDI hostname
```

Comment:
What If we updated the Background section by adding:

"Allowing a user to load variables is still a security risk, even from the same host. However the program requirements might dictate it's necessary. We're going to assume that in this exercise it's required to load variables from a JNDI server on the same host, and disabling such functionality is not an option."

Reply:
That makes sense to me!

Reply:
However, it's not really dealing with the underlying issue I raised earlier.